app/views/passengers/_form.html.haml

```diff
@@ -19,22 +25,32 @@
     = form.label :phone_number
     = form.phone_field :phone
   .field
+    = form.label :umass_status
+    = form.select :status, Passenger::STATUSES, include_blank: true
+  .field
+    = form.label :registered_with_disability_services
+    = form.check_box :registered_with_disability_services,
+      id: "pax_registered_with_DS"
+  .field
+    = form.label :has_brochure
+    = form.check_box :has_brochure,
+      id: "pax_has_brouchure"
+  .field
     = form.label :mobility_device_id
     = form.select :mobility_device_id, options_from_collection_for_select(MobilityDevice.order(:name), :id, :name), include_blank: 'None'
   - if @current_user.admin?
     .field
       = form.label :active
       = form.check_box :active
-  .field
```

Comment:
Why remove this?

Reply:
I moved it to the top. Checking it causes a couple of boxes that are only relevant to temporary passengers to deactivate, so I thought putting it further up made the most sense.